package/libyui-qt.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Sun Feb  5 14:43:49 CET 2017 - besser82@fedoraproject.org
+
+- Fix crash when shutting down the UI
```

Comment:
please reference here bug reports like `(gh#libyui-libyui#41, bsc#931762)` so we have reference what problems we are fixing. Thanks and same for hanging if such issue exists.